### ISSUE TITLE
Add Cohort Type - Proper error message 

### ISSUE THREAD

**Issue by zohaibihs** [OpenSRP/openmrs-module-cohort#16]
Proper error message like Cohort Name is required and Cohort Description is required should be appeared when no data is filled.(UI Suggestion - Low) (The same bug should be fixed on all the other modules).

![capture](https://cloud.githubusercontent.com/assets/20788534/17326390/7d9140fa-58ca-11e6-821c-fb9e07cd8e4a.PNG)

**Comment by bholagabbar** [comment]
This is probably on some older version. It has been fixed quite a while back now. Here is a screenshot.
![capture2](https://cloud.githubusercontent.com/assets/11693595/17743685/2d11522e-64c2-11e6-8015-57dc9d901882.PNG)

**Comment by zohaibihs** [comment]
new version 2.0.0 same issue faced.
Proper error message like Cohort Name is required and Cohort Description is required should be appeared when no data is filled.(UI Suggestion - Low) (The same bug should be fixed on all the other modules).

![7d9140fa-58ca-11e6-821c-fb9e07cd8e4a](https://cloud.githubusercontent.com/assets/20788534/17806506/730683dc-661e-11e6-8a6e-b6d35f5e60b4.PNG)

![image](https://cloud.githubusercontent.com/assets/20788534/17806601/f1aad7a6-661e-11e6-84cb-280aa1202dec.png)

![image](https://cloud.githubusercontent.com/assets/20788534/17806613/f9c9265e-661e-11e6-9ca8-a6ecccadaf6e.png)

**Comment by bholagabbar** [comment]
Yes I notices this. If you run it locally, it only comes 'required' I have
no clue why @maimoonak
